lib/modules/keyboardNav.js

```diff
@@ -1104,6 +1113,12 @@ function reply(selected = getSelected()) {
 	));
 }
 
+function edit(selected = getSelected()) {
+	if (selected.isComment() || selected.isSelfPost()) {
+		click(getFirstElementInThingByQuery('.edit-usertext', ASSERT, selected));
```

Comment:
`.entry .edit-usertext` so that child it doesn't check child comments (in case `e` is pressed on a non-editable thing).